src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+// import cn from 'classnames';
+
+export const Person = ({ person: {
+  name,
+  age = 0,
+  sex,
+  isMarried,
+  partnerName,
+} }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+
+    { age !== 0 && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      { (sex === 'm' && (isMarried)) && (
+        `${partnerName} is my wife`
+      )}
+
+      { (sex === 'f' && (isMarried)) && (
+        `${partnerName} is my husband`
```

Comment:
Let's create a variable for wife/husband status to simplify the conditions
![image](https://github.com/mate-academy/react_person/assets/96409563/20aa880f-67cf-440e-b277-59b9564cf660)